src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.vb

```diff
@@ -5639,6 +5639,25 @@ Imports TestAssembly.BothObsoleteParent.BothObsoleteChild
 BC40008: 'BothObsoleteParent.BothObsoleteChild' is obsolete.
 Imports TestAssembly.BothObsoleteParent.BothObsoleteChild
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+]]></expected>)
+        End Sub
+
+        <Fact>
+        <WorkItem(19394, "https://github.com/dotnet/roslyn/issues/19394")>
+        Public Sub WellKnownTypeAsStruct_DefaultConstructor_ParamArrayAttribute()
+            Dim code = <compilation><file name="a.vb"><![CDATA[
+Namespace System
+	public Structure ParamArrayAttribute
```

Comment:
>ParamArrayAttribute [](start = 18, length = 19)

It would be good to have test targeting usage on non-default struct constructor. Perhaps with tuples?